sbi/inference/posteriors/ratio_based_posterior.py

```diff
@@ -325,6 +319,50 @@ def _num_trained_rounds(self, trained_rounds: int) -> None:
             f"can evaluate the {normalized_or_not}posterior density with .log_prob()."
         )
 
+    @staticmethod
+    def _log_ratios_over_trials(
+        x: Tensor,
+        theta: Tensor,
+        net: nn.Module,
+        track_gradients: bool = False,
+    ) -> Tensor:
+        """Return log ratios summed over iid trials of x.
```

Comment:
Please synchronise docstring with current version of the snle docstring (this one is deprecated, it misses ``)